mxfusion/components/distributions/random_gen.py

```diff
@@ -83,6 +91,20 @@ def sample_multinomial(data, get_prob=True, dtype='int32', F=None):
         return F.random.multinomial(
             data=data, get_prob=get_prob, dtype=dtype)
 
+    @staticmethod
+    def sample_bernoulli(prob_true=0.5, dtype='bool', shape=None, F=None):
+        """
+        Sample Bernoulli distributed variables
+
+        :param shape: Array shape of samples
+        :param prob_true: Probability of being true
+        :param dtype: data type
+        :param F: MXNet node
+        :return: Array of samples
+        """
+        F = get_default_MXNet_mode() if F is None else F
+        return F.random.uniform(low=0, high=1, shape=shape) > prob_true
```

Comment:
Do you need to pass dtype in anywhere here? Is it useful to have it as a parameter even if the outcome is a boolean array?

Reply:
Done, and added test